service.go

```diff
@@ -355,12 +362,19 @@ func InitService() {
 	LocalPodIp = os.Getenv("POD_IP")
 	LocalPodName = os.Getenv("POD_NAME")
 	S3BucketForTiFLashLog = os.Getenv("S3_FOR_TIFLASH_LOG")
-	CheckTiFlashFreeTimeoutString := os.Getenv("CHECK_TIFLASH_FREE_INTERVAL")
+	CheckTiFlashFreeTimeoutString := os.Getenv("CHECK_TIFLASH_FREE_TIMEOUT")
```

Comment:
use var declare instead to remove magic number "CHECK_TIFLASH_FREE_TIMEOUT" and "CHECK_TIFLASH_FREE_INTERVAL"